src/store/initial-state.js

```diff
@@ -64,14 +70,42 @@ export const mergeLocalStorage = (state) => {
  * @param {object} data Data prop passed to App component
  * @param {boolean} applyFixes Whether to override initialState
  */
-export const preparePipelineState = (data, applyFixes) => {
+export const preparePipelineState = (data, applyFixes, expandAllPipelines) => {
   const state = mergeLocalStorage(normalizeData(data));
   if (applyFixes) {
     // Use main pipeline if active pipeline from localStorage isn't recognised
     if (!state.pipeline.ids.includes(state.pipeline.active)) {
       state.pipeline.active = state.pipeline.main;
     }
   }
+
+  // Cater for expandAllPipelines in component props or within flag
+  // expandAllPipelines needs to happen after the deepmerge of localStorage to overwrite
+  // any saved visible state of the nodes
```

Comment:
This multi-line comment format is mainly reserved for description of functions - I've cleared up this comment though so it will now fit nicely in one line.